Security machine learning rule documentation improvements (#2240)
elastic/docs-content: solutions/security/advanced-entity-analytics/anomaly-detection.md

> # Anomaly detection

Reviewer: Whoever reviews this from my team: should we link to this page from:

- https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2240/solutions/security/explore/hosts-page,
- https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2240/solutions/security/explore/network-page
- https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2240/solutions/security/explore/users-page
?

Reviewer 2: Yeah, it would make sense to link to it from the Data tables sections on those pages.

::::{note}
[{{ml-cap}}](/explore-analyze/machine-learning/anomaly-detection.md) functionality is available when you have the appropriate role, subscription, are using a [cloud deployment](https://cloud.elastic.co/registration?page=docs&placement=docs-body), or are testing out a **Free Trial**. Refer to [Machine learning job and rule requirements](/solutions/security/advanced-entity-analytics/machine-learning-job-rule-requirements.md) for more information.
::::

> Anomaly detection jobs allow you to to identify anomalous events or patterns in your data. In a security context, they are typically used with detection rules to create alerts when there is divergence from baseline data.

Reviewer: Do they allow the user to identify or do they identify themselves?
Also do they create alerts or trigger them?

I went with my assumptions in this suggestion but I might be wrong:

```suggestion
Anomaly detection jobs identify anomalous events or patterns in your data. In a security context, they’re typically used with detection rules to trigger alerts when behavior deviates from baseline activity.
```

::::{tip}

> See [{{ml-cap}}: Anomaly detection](/explore-analyze/machine-learning/anomaly-detection.md) and [About detection rules](/solutions/security/detect-and-alert/about-detection-rules.md) for more background.

Reviewer 2: ```suggestion
Refer to [{{ml-cap}}: Anomaly detection](/explore-analyze/machine-learning/anomaly-detection.md) and [About detection rules](/solutions/security/detect-and-alert/about-detection-rules.md) for more background.
```

    :screenshot:
    :::

::::{tip}

> For an overview of creating machine learning rules, see [Create a machine learning rule](/solutions/security/detect-and-alert/create-detection-rule.md#create-ml-rule).

Reviewer 2: ```suggestion
For instructions on creating {{ml}} rules, refer to [Create a machine learning rule](/solutions/security/detect-and-alert/create-detection-rule.md#create-ml-rule).
```

## View detected anomalies [view-anomalies]

> From the security solution, you can view the details of detected anomalies within the `Anomalies` table widget shown on the Explore > Hosts, Network, and Users pages, or even narrow to the specific date range of an anomaly from the `Max anomaly score by job` field in the overview of the details pages for hosts and IPs. These interfaces also offer the ability to drag and drop details of the anomaly to Timeline, such as the `Entity` itself, or any of the associated `Influencers`.

Reviewer: It looks like we call these data tables [elsewhere](https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/2240/solutions/security/explore/hosts-page) in the docs.

What about something like this?

```suggestion
View details of detected anomalies in the **Anomalies** data tables, available on the **Hosts**, **Network**, or **Users** pages. You can access these pages from the navigation menu or with the [global search field](../../explore-analyze/find-and-organize/find-apps-and-objects.md).

You can narrow the time range to a specific anomaly using the `Max anomaly score by job` field in the overview section of the host or IP details. You can also drag and drop anomaly details, like the `Entity` or any associated `Influencers`, into **Timeline**.
```

Reviewer: Thanks for the PR! I know you didn't write this part but figured we could clean it up anyway 😄
